styles/components/app-profile.css

```diff
@@ -20,6 +20,7 @@
 }
 
 .app-profile__logout {
+  gap: 500px;
```

Comment:
that looks a lot -maybe it could cause responsive design problems - what do you think?